experiments/standalone/Snow/snowmip_simulation.jl

```diff
@@ -89,6 +110,29 @@ S = [parent(sol.u[k].snow.S)[1] for k in 1:length(sol.t)];
 U = [parent(sol.u[k].snow.U)[1] for k in 1:length(sol.t)];
 t = sol.t;
 
+start_day = 1
+days = start_day .+ floor.(t ./ 3600 ./ 24)
+doys = days .% 366 # doesn't account for leap year
+
+obs_swes = Vector{Union{Float64, Missing}}(missing, length(doys))
+for (i, avail) in enumerate(snow_data_avail)
+    if avail
+        # obs_swes[i] = z[i] * (mass[i] / z[i]) / 1000.0
+        obs_swes[i] = mass[i] / 1000.0
```

Comment:
I think this is fine, but we also have this already https://github.com/CliMA/ClimaLand.jl/blob/96ee0b973362d8ec42188c283b30be91aeb6faa8/experiments/standalone/Snow/process_snowmip.jl#L126C1-L126C45

Reply:
You may be able to shorten this like:
```
obs_swes[snow_data_avail] .= mass[snow_data_avail] ./ 1000
```
because I believe snow_data_avail is a mask. This would be a bit more readable I think